.github/workflows/android_qt6.yaml

```diff
@@ -0,0 +1,67 @@
+name: Android (QT6)
+on:
+  push:
+    branches:
+      - main
+      - 'releases/**'
+  pull_request:
+    branches:
+      - main
+      - 'releases/**'
+
+
+jobs:
+  compile-android-qt6:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false # Don't cancel other jobs
+      matrix:
+        config:
+            #- { name: Release, args: "", apk: .tmp/src/android-build/build/outputs/apk/release/, sym: true, sign: true  }
+            - { name: Debug, args: "--debug", apk: .tmp/src/android-build/mozillavpn.apk, sym: false, sign: false}
+        arch: 
+          # Qt6 names their arch diffrent then what qmake expects ...
+          - {name: "armv7", qmake: "armeabi-v7a"}
+          - {name: "arm64_v8a", qmake: "arm64-v8a"}
+          - {name: "x86" , qmake: "x86"}
+          - {name: "x86_64", qmake: "x86_64"}
+        version: ["6.2.1"]
+    container:
+      image: ghcr.io/strseb/vpn-builder-android:android_${{matrix.arch.name}}_${{matrix.version}}
```

Comment:
Since i guess our end-goal is to move all build steps to task cluster, I figured I might as well write a docker image to build it :) - as we would need to do this anyway at some point. 
currently lives here: https://github.com/strseb/vpn-buildcontainer-android
next step after this pr would probably be to move this container into the vpn-repo aswell.